orchestrator/db/models.py

```diff
@@ -418,6 +418,9 @@ def select() -> Select:
             select(WorkflowTable).options(undefer(WorkflowTable.deleted_at)).filter(WorkflowTable.deleted_at.is_(None))
         )
 
+    def is_task(self) -> bool:
+        return self.target == Target.VALIDATE
```

Comment:
Both Validation and System workflows are tasks :) 

```suggestion
    def is_task(self) -> bool:
        return self.target in (Target.VALIDATE, Target.SYSTEM)
```

Reply:
Got it!